homeassistant/helpers/entity.py

```diff
@@ -93,7 +93,7 @@ def should_poll(self) -> bool:
 
     @property
     def unique_id(self) -> str:
```

Comment:
Unrelated, but why should `unique_id` return a `str` when it is clearly a `Optional[str]`? I mean we should Indi ate that a `None` return value is also acceptable.

Reply:
Correct